.github/workflows/spelling.yml

```diff
@@ -0,0 +1,25 @@
+name: spellcheck
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  spelling:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
I think Github will complain about the version of checkout, latest version is `v3`.